libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -623,52 +624,61 @@ static CK_RV prvCertAttParse( CK_ATTRIBUTE * pxAttribute,
         case ( CKA_VALUE ):
             *ppxCertificateValue = pxAttribute->pValue;
             *pxCertificateLength = pxAttribute->ulValueLen;
+            LogDebug( ( "Found CKA_VALUE attribute. Certificate value length was: %lu.", pxAttribute->ulValueLen ) );
             break;
 
         case ( CKA_LABEL ):
+            LogDebug( ( "Found CKA_LABEL attribute." ) );
 
             if( pxAttribute->ulValueLen <= pkcs11configMAX_LABEL_LENGTH )
             {
                 *ppxLabel = pxAttribute;
             }
             else
             {
-                LogError( ( "Label length was not in the valid range. Please check the value of pkcs11configMAX_LABEL_LENGTH" ) );
+                LogError( ( "Failed parsing certificate template. Label length"       \
+                            "was not in the valid range. Found %lu and expected %lu." \
+                            "Consider updating pkcs11configMAX_LABEL_LENGTH.", pxAttribute->ulValueLen, pkcs11configMAX_LABEL_LENGTH ) );
```

Comment:
Backslash escapes are only requires for macro definitions, and not for macro calls
```suggestion
               LogError( ( "Failed to parse certificate template. Label length "
                "not in valid range. Found %lu and expected %lu. "
                "Consider updating pkcs11configMAX_LABEL_LENGTH.",
                pxAttribute->ulValueLen, pkcs11configMAX_LABEL_LENGTH ) );
```